test/OpenTelemetry.Tests/Metrics/MetricApiTestsBase.cs

```diff
@@ -1613,6 +1613,34 @@ public void UnsupportedMetricInstrument()
         Assert.Empty(exportedItems);
     }
 
+    [Fact]
+    public void GaugeIsExportedCorrectly()
```

Comment:
Please expand the test to cover both temporalities. Though Gauge has no temporality notion like other instruments, but still temporality affects if previous reported metricpoints are remembered/forgot after collect.

In particular, we need a test to validate that
For sync gauge, if cumulative temporality is set, metrics are exported every collect, irrespective of whether new measurements were reported or not.

Reply:
Relevant spec wording : https://github.com/open-telemetry/opentelemetry-specification/pull/3540/files#diff-32d5f289beed96900fa1e6e69a3ee9cc0b6503ce8883672011c44bc772c0fa2aR1099

Reply:
Fixed, please check.